### ISSUE TITLE
Add descriptions to variables

### ISSUE THREAD

**Issue by TheInvoker** [a85/POSTMan-Chrome-Extension#11303]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Yes, I want to be able to add comments to variables/global variables, but there is no column for this currently.

### Describe the solution you'd like

A new column option for comments.

**Comment by DannyDainton** [comment]
Hey @TheInvoker

Could you clarify what you mean by 'Comments', please?

We do have a comments feature which allows you to leave them at different levels (Collections, Folders, Requests) and on different elements (Params, Headers, Scripts, etc)

https://learning.postman.com/docs/collaborating-in-postman/working-with-your-team/discussing-your-work/#contents

Are you after adding descriptions to variables instead?

**Comment by TheInvoker** [comment]
@DannyDainton Yes exactly. I have a bunch of variables defined and need to add a description to them to remember things. But the only columns are Variable, Initial Value, Current Value. If I click the 3 dots, I can only remove initial value and current value. But it would be super helpful to have another column (can be unchecked by default) that can store some description.

Its inconsistent that I can add description to like header or param values, but not variables.

Currently I need this for the collection variables, but would be great if it can be added to globals and environment variables as well.

**Comment by JanDoggen** [comment]
Yes, tt would be really helpful if we could add descriptions to environment variables.

**Comment by natebgurutech** [comment]
Any news on this? I would really love to have this. Usually in a `.env` file we have something like 
```
# This is the secret key for the foo
FOO=123
```

which lets us remember what all of these variables are. Of course, we should name variables where they don't need a comment in all situations possible, but sometimes it's just not feasible in environment variables.

**Comment by zhaowb** [comment]
My bypass currently is right under the variable add another variable with same name plus '_comment' suffix, and the value is description.
It looks ugly but better than nothing.

**Comment by swinster** [comment]
Also, see https://github.com/postmanlabs/postman-app-support/issues/10217

**Comment by swinster** [comment]
FWIW, I was using the "Initial Value" filed for a description, which is perhaps even uglier than @zhaowb 's method :(

![image](https://github.com/postmanlabs/postman-app-support/assets/8142964/7884c2ec-1485-4e59-9f9b-b3ad28764cfe)

**Comment by swinster** [comment]
Also see https://github.com/postmanlabs/postman-app-support/issues/5976

**Comment by swinster** [comment]
Also https://github.com/postmanlabs/postman-app-support/issues/4030

**Comment by rvq** [comment]
Really looking forward for this feature

**Comment by KevinNovak** [comment]
+1 looking forward to this. Especially now with the Type: secret, it's hard to tell if a variable value has some comments in it

**Comment by Threnos** [comment]
+1 Would be helpful to write notes to variables. Just to remember at a glance what that id (that has same name between environments) is for and what value it holds.

**Comment by giridharvc7** [comment]
Thank you for raising this concern, we are working on this feature. 

Since there are many requests for this feature, we are gonna track this under #5977. So closing this issue as duplicate

**Comment by skatikia** [comment]
@TheInvoker Good news! With the latest update to the variables interface (Postman V11.64.0), you can now add descriptions to variables. 🎉

Read all about the latest changes to the variables interface: https://blog.postman.com/engineering/a-simpler-and-local-first-variables-experience/

![Image](https://github.com/user-attachments/assets/2d1bd9ef-4486-439b-a6b0-5e610244fcf3)
